### PR TITLE
Added orbit template parameters into the orbit response

### PR DESCRIPTION
**Pull request by Tasiobg**
Added orbit template parameters into the orbit response so we can extract them from a response and use it in unitest

### CODE REVIEW

**Comment by BPScott** [comment]
Add a unit test for passing in some values to this. I'm guessing the current test is failing as you've changed the default response.

Would it be worth doing the same thing for `requestParams` in case we need to test them too at a later date?

**Comment by Tasiobg** [comment]
@BPScott I have added a unitest, please check again.

Not sure if is worth to add `$requestParams`, this is an example of the content
```
  'variant' => 'default'
  'language' => 'en-GB' 
```
What do you think? should I add it?

**Comment by BPScott** [comment]
I think it would be useful to prove that those values are being passed in as part of some integration tests. Let's add an element for that too

File: tests/OrbitStubClientTest.php [hunk 1]
```diff
@@ -29,11 +29,25 @@ public function testGetContentWithPresentButUnusedConstructor()
...
         $this->assertEquals($expectedOrbit, $orbitClient->getContent());
     }
+
+    public function testGetContentWithTemplateParameters()
+    {
+        $orbitClient = new OrbitStubClient();
+
+        $expectedOrbit = new Orbit(
+            '<orbit-head><orbit-template-params data-values="'
+                    . htmlentities(json_encode(['testKey' => 'testValue'])) .'"/></orbit-head>',
```
Comment (BPScott) [inline tests/OrbitStubClientTest.php]: Would it be better to be explicit with our expected output ` <orbit-head><orbit-template-params data-values="[&quot;testKey&quot;=>&quot;testValue&quot;]"/></orbit-head>`?

Then we can be sure we're getting the right output. At the moment you're testing `htmlentities(json_encode(['testKey' => 'testValue']))` is the same output as `htmlentities(json_encode(['testKey' => 'testValue']))` which doesn't seem overly helpful :)

Comment (Tasiobg) [inline tests/OrbitStubClientTest.php]: no preference about this point, I thought the first one was not easy to read so I decided to use the encoding functions. I'm going to update it with the raw value, thanks for the suggestion

**Pull request merged**